ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master__4.15.yaml

```diff
@@ -558,6 +558,7 @@ tests:
     env:
       BYO_OIDC: "true"
       CHANNEL_GROUP: stable
+      ENABLE_BILLING_ACCOUNT: "yes"
```

Comment:
I see that we are "locked" onto 4.15 for the e2e-rosa tests.
Is it because it is only supported on 4.15? Or is it to reduce number of tests (reduce costs)?